_includes/templates/corp-info-org.html

```diff
@@ -0,0 +1,63 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of social media links
++
++ Used by organizational and institutional page
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-crprt">
```

Comment:
We will probably need to move the associated style of `gc-crprt` either with its associated template or as an independent component (like when it's reuse by multiple template).

see: https://github.com/wet-boew/GCWeb/blob/master/components/_base.scss#L57

We might reconsider to rename this file too. The goal is by their location and by their file name we can easily link them together without deep diving into the code and without causing any breaking change according to our public versioning API. (here the CSS class name that was used). 

FYI - We have the following word in this context which might require some adaptation : "corp-info-org" + "gc-crprt" + "Organizational/Institutional template" + "Social media links" (as line 4) + "Corporate information"